src/test/kotlin/com/zepben/evolve/streaming/data/SetCurrentStatesStatusTest.kt

```diff
@@ -82,7 +83,7 @@ class SetCurrentStatesStatusTest {
     }
 
     @Test
-    fun `StateEventFailure from protobuf and then back to protobuf`() {
+    internal fun `StateEventFailure from protobuf and then back to protobuf`() {
         val unknowMridPb = PBStateEventFailure.newBuilder().apply {
```

Comment:
while you're here, fix this typo

Reply:
Fixed in c2bbd5d5 ad subsequently removed as part of rework in e6fa0424